orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -73,6 +73,20 @@ def bert_model_description(dynamic_shape=True):
 def optimizer_parameters(model):
     '''A method to assign different hyper parameters for different model parameter groups'''
 
+    no_decay_keys = ["bias", "gamma", "beta", "LayerNorm"]
+    no_decay_param_group = []
+    decay_param_group = []
+    for initializer in model.graph.initializer:
+        if any(key in initializer.name for key in no_decay_keys):
+            no_decay_param_group.append(initializer.name)
+        else:
+            decay_param_group.append(initializer.name)
+    params = [{'params': no_decay_param_group, "alpha": 0.9, "beta": 0.999, "lambda_coef": 0.0, "epsilon": 1e-6},
+              {'params': decay_param_group, "alpha": 0.9, "beta": 0.999, "lambda_coef": 0.01, "epsilon": 1e-6}]
+    return params
+
+def optimizer_parameters_old(model):
+    '''A method to assign different hyper parameters for different model parameter groups'''
```

Comment:
```suggestion
    '''A method to assign different hyper parameters for different model parameter groups'''

```
Always add an empty line after docstring